integration/auth/auth_test.go

```diff
@@ -57,6 +58,7 @@ func TestMain(m *testing.M) {
 	if err != nil {
 		log.Fatalln(err)
 	}
+	rand.Seed(time.Now().UTC().UnixNano())
```

Comment:
In tests that use random, it's a good practice to log the seed passed to the randomizer, so that if the test turns out to be flaky, you can replay the failed test run exactly by reusing the seed.

Reply:
Good idea. Done.